scripts/convert-to-geomdat/convert-to-geomdat.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+
+# This script automatically converts a restart file to a Geometry.dat with AIMS runs.
+# It assumes that you have velocities in the restart file.
+
+list = [x for x in range(1)]
+for i in list:
+    atomnum = 0
+    atomcoord = np.array([])
+    atomvel = np.array([])
+    namelist = []
+    atommass = np.array([])
+    with open("./" + str(i) + "coors.rst7", "r") as f:
```

Comment:
Script is broken here. These files do not exist. We have hard-coded strings, and are using non-windows compatible stings as file paths. 

Do we want to refactor this code to be a working script? Or just remove it? We don't want to just collect old broken scripts in our repo ;P (obviously!). So we need to make sure they work, or not have them here, I think.

Reply:
I mean refactoring would be cool, but I was under the impression we should add things people regularly use to /scripts that can be refactored to be included in AIMS later and/or just used as-is. 

This particular script (for example) is what people usually use to convert their terachem coordinate files to AIMS Geometry.dat files. So the functionality is important. I wouldn't call it "broken" per se

Reply:
err aimsprop not AIMS